pallets/subtensor/src/epoch/math.rs

```diff
@@ -113,26 +113,24 @@ pub fn vec_max_upscale_to_u16(vec: &[I32F32]) -> Vec<u16> {
                     })
                     .collect();
             }
-            return vec
-                .iter()
+            vec.iter()
                 .map(|e: &I32F32| {
                     e.saturating_mul(u16_max)
                         .saturating_div(*val)
                         .round()
                         .to_num::<u16>()
                 })
-                .collect();
+                .collect()
         }
         None => {
             let sum: I32F32 = vec.iter().sum();
-            return vec
-                .iter()
```

Comment:
Do not change the math functions. they have been battled tested

Reply:
This change from clippy. Without it you can't pass CI.

Reply:
+1

Reply:
Or, well, you can by disabling the lint. But is it a big deal to remove `return` and return the value in a rusty way?

Reply:
no no its ok , just make sure tests pass. Its from 1.83 , and lints probably got stricter